scripts/play/index.js

```diff
@@ -55,9 +56,15 @@ function run() {
 	}
 
 	Object.assign(window, { ...fenFuncs });
+
 	setupBoard();
-	newBoard(8, true);
-	alignBoard();
+	if (window.gameOptions.puzzles) {
+		getPuzzles().then(() => setBoard(0));
```

Comment:
To set the mode of the game
```suggestion
		$('body').dataset.mode = 'puzzles';
		getPuzzles().then(() => setBoard(0));
```